dist/tools/flake8/check.sh

```diff
@@ -43,7 +45,19 @@ ${FLAKE8_CMD} --version &> /dev/null || {
     exit 1
 }
 
-ERRORS=$(${FLAKE8_CMD} --config="${RIOTTOOLS}"/flake8/flake8.cfg ${FILES})
+ERRORS=$(${FLAKE8_CMD} --config="${RIOTTOOLS}"/flake8/flake8.cfg ${FILES} | ${LOG})
+
+if github_annotate_is_on; then
+    grep "^.\+:[0-9]\+:" "${LOGFILE}" | while read line; do
+        FILENAME=$(echo "${line}" | cut -d: -f1)
+        LINENR=$(echo "${line}" | cut -d: -f2)
+        DETAILS=$(echo "${line}" | cut -d: -f3- |
```

Comment:
From the output of the test annotation: I think `-f3` is the column (which sadly, as far as my trials could gather, can't be annotated with `github_annotate_error`). Rather do

```suggestion
        DETAILS=$(echo "${line}" | cut -d: -f4- |
```